### PR TITLE
Permit merra to use the DB inventory, and fix a prism tab bug

### PR DESCRIPTION
**Pull request by bhbraswell**
Unless I'm missing something, prism driver can't work in dev because of a bad line spacing. This change seems to make it OK for me.

### CODE REVIEW

**Comment by bhbraswell** [comment]
Oh yeah also merra is not busted. That commit is in there by accident but should still be OK to have in dev.

File: gips/inventory/orm/__init__.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def setup():
     if setup_complete:
         return
     if use_orm():
-        busted_drivers = ('sar', 'sarannual', 'modaod', 'merra', 'daymet', 'cdl')
+        busted_drivers = ('sar', 'sarannual', 'modaod', 'daymet', 'cdl')
```
Comment (ags-tolson) [inline gips/inventory/orm/__init__.py]: To my shame I didn't document why I blacklisted merra in the first place, but would you mind describing how you know merra is safe?  Like what testing you may've done, etc?

Comment (bhbraswell) [inline gips/inventory/orm/__init__.py]: sure
```
gips_project merra -p tmin tmax srad prcp wind -d 2014,2016 -s gips/gips/test/NHseacoast.shp --alltouch --fetch --outdir merra_test -v4
```
takes forever, but works

Comment (ags-tolson) [inline gips/inventory/orm/__init__.py]: Okay, presuming that's done with `export GIPS_ORM=true`, and presuming `gips_inventory merra` looks right afterwards, I think this is ready to merge :+1:

**Comment by ags-tolson** [comment]
This PR has been superseded by #250

**Pull request closed without merging**